### PR TITLE
Add time range tests.

### PR DESCRIPTION
**Pull request by andrey-star**

### CODE REVIEW

File: step-release-vis/src/app/components/environment/environment_test.ts [hunk 1]
```diff
@@ -410,15 +413,52 @@ describe('EnvironmentComponent', () => {
...
+      const svg = fixture.debugElement.query(
+        By.css(`#${component.environment.name}-svg`)
+      );
+      spyOn(component.startTimestampChange, 'emit');
+      spyOn(component.endTimestampChange, 'emit');
+
+      svg.triggerEventHandler('mousedown', {pageX: 100, pageY: 100});
+      svg.triggerEventHandler('mousemove', {pageX: 200, pageY: 200});
+      component.draggedEnvName.name = 'other_env';
+
```
Comment (mracz) [inline step-release-vis/src/app/components/environment/environment_test.ts:441]: I do not see mouseup here, is it the test, the test description or implementation which is wrong? :D

File: step-release-vis/src/app/components/environment/environment_test.ts [hunk 2]
```diff
@@ -410,15 +413,52 @@ describe('EnvironmentComponent', () => {
...
   describe('time range drag', () => {
     it('should update time range', () => {
-      // TODO(#277): implement test
+      const svg = fixture.debugElement.query(
+        By.css(`#${component.environment.name}-svg`)
+      );
+      spyOn(component.startTimestampChange, 'emit');
+      spyOn(component.endTimestampChange, 'emit');
+      svg.triggerEventHandler('mousedown', {pageX: 100, pageY: 100});
+      fixture.detectChanges();
```
Comment (mracz) [inline step-release-vis/src/app/components/environment/environment_test.ts:422]: Could you add this to a separate test, so we don't have multiple expect phases in a single test?
Also please add a blank line between setup/action/expect (as you did below)

File: step-release-vis/src/app/components/environment/environment_test.ts (hunk 1 above)
Comment (andrey-star) [reply step-release-vis/src/app/components/environment/environment_test.ts:441]: The mouseup is supposed to happen in a sibling environment, can't do that explicitly in this test. I'll update the description.

File: step-release-vis/src/app/components/environment/environment_test.ts (hunk 2 above)
Comment (andrey-star) [reply step-release-vis/src/app/components/environment/environment_test.ts:425]: Done

**Review by mracz — APPROVED** [review]

**Pull request merged**